src/vue/components/panel-compact.vue

```diff
@@ -268,107 +323,80 @@ export default {
     infoIcon: InfoIcon,
     grammarIcon: GrammarIcon,
     treebankIcon: TreebankIcon,
+    wordUsageIcon: WordUsageIcon,
     alphTooltip: Tooltip,
     lookup: Lookup,
-    reskinFontColor: ReskinFontColor
+    reskinFontColor: ReskinFontColor,
+    wordUsageExamplesBlock: WordUsageExamplesBlock
   },
   directives: {
     onClickaway: onClickaway
   },
+  positionClassVariants: {
+    left: 'alpheios-panel-left',
+    right: 'alpheios-panel-right'
+  },
+  minWidth: 400, // A minimal width of a panel, in pixels
+  defaultScrollPadding: 20,
   data: function () {
     return {
       inflectionsPanelID: 'alpheios-panel__inflections-panel',
       inflectionsBrowserPanelID: 'alpheios-panel__inflections-browser-panel',
-
-      positionClassVariants: {
-        left: 'alpheios-panel-left',
-        right: 'alpheios-panel-right'
-      },
-
+      panelPosition: 'left',
       panelLeftPadding: 0,
       panelRightPadding: 0,
       scrollPadding: 0,
-      defaultScrollPadding: 20,
-      defaultPanelWidth: 400,
-      panelWidth: null
+      panelWidth: null,
+      styles: {
+        zIndex: this.ui.zIndex
+      }
     }
   },
   props: {
     data: {
       type: Object,
       required: true
-    },
-    classesChanged: {
-      type: Number,
-      required: false,
-      default: 0
     }
   },
 
   computed: {
     divClasses () {
-      return (this.data && this.data.classes ? this.data.classes.join(' ') : '') + ' ' + this.positionClasses
+      return this.data.classes.concat([this.$options.positionClassVariants[this.panelPosition]])
     },
     clearLookupText: function () {
       // always true to clear panels lookup
       return true
     },
     lookupParentLanguage: function () {
-      if (this.data.infoComponentData) {
-        return this.data.infoComponentData.languageName
+      if (this.$store.state.app.currentLanguageName) {
+        return this.$store.state.app.currentLanguageName
       } else {
-        return this.options.items.preferredLanguage.currentTextValue()
+        return this.settings.contentOptions.items.preferredLanguage.currentTextValue()
       }
     },
     inflectionBrowserLanguageID: function () {
-      return this.data.currentLanguageID
-    },
-    uiController: function () {
-      return (this.$parent && this.$parent.uiController) ? this.$parent.uiController : null
+      return this.$store.state.app.currentLanguageID
```

Comment:
can't this be moved inside the inflection-browse component?

Reply:
Yes, that's a great suggestion, will do it.